model/roles_test.go

```diff
@@ -1232,8 +1232,28 @@ func TestLoadRoleManifestColocatedContainersValidationPortCollisions(t *testing.
 	roleManifestPath := filepath.Join(workDir, "../test-assets/role-manifests/model/colocated-containers-with-port-collision.yml")
 	roleManifest, err := LoadRoleManifest(roleManifestPath, []*Release{torRelease, ntpRelease}, nil)
 	assert.Nil(roleManifest)
-	assert.EqualError(err, "role[main-role]: Invalid value: 80: port collision, the same port is used by: main-role, to-be-colocated\n"+
-		"role[main-role]: Invalid value: 10443: port collision, the same port is used by: main-role, to-be-colocated")
+	assert.EqualError(err, "role[main-role]: Invalid value: \"TCP/10443\": port collision, the same protocol/port is used by: main-role, to-be-colocated"+"\n"+
+		"role[main-role]: Invalid value: \"TCP/80\": port collision, the same protocol/port is used by: main-role, to-be-colocated")
```

Comment:
Ah, the sorting is off because it's alpha-sort, not numeric. That should be okay :D

Reply:
That swap annoyed me a little bit. But on the other side, keeping the numeric sorting would be possible, but would require a Golang version 1.8 feature to sort with a custom sorting function. Since this is only for the error use case, I figured one could live with that missing numeric sort order. Let me know, if you would like it any other way.